src/components/Adyen/Adyen.js

```diff
@@ -121,7 +123,9 @@ const Adyen = ({
     if (containerRef.current) {
       const dropin = checkout.create('dropin', {
         onSelect,
-        openFirstPaymentMethod: !window.matchMedia('(max-width:991px)').matches
+        openFirstPaymentMethod:
+          adyenConfiguration?.openFirstPaymentMethod ||
+          !window.matchMedia('(max-width:991px)').matches
```

Comment:
What in case if publisher will pass `false` as openFirstPaymentMethod? Then we will be checking the max-width property